services/libs/opensearch/src/repo/indexing.repo.ts

```diff
@@ -19,11 +19,11 @@ export class IndexingRepository extends RepositoryBase<IndexingRepository> {
     )
   }
 
-  public async markEntitiesIndexed(type: IndexedEntityType, data: IEntityData[]): Promise<void> {
+  public async markEntitiesIndexed(type: IndexedEntityType, data: string[]): Promise<void> {
```

Comment:
why changing this to string?

Reply:
Previously, we used an IEntityData array with memberId and tenantId, but with tenantId deprecated in `indexed_entities`, we now pass only memberIds.